apps/ideas/forms.py

```diff
@@ -384,7 +365,7 @@ def save(self, commit=True):
         self.instance.collaborators.remove(*collaborators)
 
         self.instance.ideainvite_set.exclude(
-            email__in=self.cleaned_data.get('invites', [])
```

Comment:
I think I made the save, but also the clean method much harder to understand, because the collaborators and invites can now only be distinguished, because the collaborators have usernames and the invited collaborators have email-addresses. This doesn't seem like the best and a very safe idea to me, but I don't know, how to do it in a better way.

Reply:
I would recommend to somehow distinguish collaborators and invites when providing them as option. Right now you can run into the issue that a collaborator might have the email address of an invite as username.